### ISSUE TITLE
Get StockItem label

### ISSUE THREAD

**Issue by elmo2k3** [inventree/inventree-python#44]
Hey!
It would be really great if the StockItem labels could be downloaded by api.

Cheers
Bjoern

**Comment by eeintech** [comment]
As far as I can see, `StockItemLabel` data is not made available through the DRF, so this should have its own issue on the main InvenTree repo. Thoughts @SchrodingersGat ?

**Comment by SchrodingersGat** [comment]
Hi @elmo2k3 an interesting question. The label interface needs a *lot* of work, it is still very much in its infancy. Allowing access via the API would be a good idea.

How are you currently making use of the label generation functionality?

**Comment by elmo2k3** [comment]
I have added a stock item label template using the admin interface and I am using a simple css file to customize the label size (dirtily static called with LabelWriter). For printing I have an old, repaired, cab a2+.

**Comment by SchrodingersGat** [comment]
Check out https://github.com/inventree/InvenTree/pull/1212

This allows printing of labels (to PDF) via the API, like:

`/api/label/stock/20/print/?items[]=10&items[]=20`

**Comment by SchrodingersGat** [comment]
Once the linked PR is merged, we can start work on expanding the python API library to allow printing of labels to PDF
